### DISCUSSION TITLE
Important Links for reference

### DISCUSSION THREAD

**Discussion by dhairyaj** [Documentive/TemplateBuddy#1] [Ideas]
https://jsonresume.org/ --> JSON Resume Schema
Website for reference --> https://rxresu.me/
Github repo for reference --> https://github.com/AmruthPillai/Reactive-Resume
Code Style Reference Repo --> https://github.com/frankhart2018/cs-5610-404-not-found

Framework --> React
CSS -> Tailwind CSS + Material UI
Backend --> Node
DB --> Mongo

Initial make it complete on local, then host on Netlify(https://www.netlify.com/) + Render(https://render.com/)

Resume for reference (initial design)
[Siddhartha_Dhar_Choudhury_Resume.pdf](https://github.com/Documentive/TemplateBuddy/files/11186161/Siddhartha_Dhar_Choudhury_Resume.pdf)

**Comment by dhairyaj** [comment]
We can start with the following design and work up the UI later if needed. Ignore the greyish area. White is the whole screen.
![image](https://user-images.githubusercontent.com/31620568/231110043-4bafeb5c-ff40-499b-a0f0-1b8341a8dbc2.png)

**Comment by dhairyaj** [comment]
We should proceed with Tailwind CSS instead of Material UI. It seems easier. We can discuss this offline if needed :)

**Reply by frankhart2018** [reply]
I haven't used tailwind, mui is pretty simple tbh. I am fine with anyone 😁

**Comment by dhairyaj** [comment]
For Icons, we will use the package react-icons (https://react-icons.github.io/react-icons/) --> has icons from a lot of different iconpacks

**Reply by dhairyaj** [reply]
Added material ui icons as well. might remove the react-icons pack if not used further

**Comment by frankhart2018** [comment]
Generate PDFs by dumping them into LaTeX templates and generating PDFs from that. For compiling LaTeX we can use: https://manuels.github.io/texlive.js/
